### DISCUSSION TITLE
west.yml--how to pass local path to repository?

### DISCUSSION THREAD

**Discussion by enoryw** [zephyrproject-rtos/zephyr#74276] [Q&A]
How to pass local path to git repo in west.yml? 

I'm using this [fork of zmk](https://github.com/moergo-sc/zmk)--I use my private repo that rebases to upstream and is accessible via ssh. Following the instructions to "local build in a container" in this [project](https://github.com/jjangga0214/zmk-config-adv360-pro), a [Docker container is created and then west.yml is copied over](https://github.com/jjangga0214/zmk-config-adv360-pro/blob/V2.0/Dockerfile) and west initializes fetching for the [public repository](https://github.com/jjangga0214/zmk-config-adv360-pro/blob/V2.0/config/west.yml).

My private repo's updated fork of zmk is accessible via ssh but since it's cloned in the Docker container I'm not sure if there's an ideal way to make it SSH-accessible. I think it makes more sense to mount it in the Docker container anyway but I've been unsuccessful trying to make west.yml use a repo providing it a local path.

**Comment by erwango** [comment]
You can add your local repo as a remote and then add remote to the manifest as done [here](https://github.com/nrfconnect/sdk-nrf/blob/8ffbc4a6af67255c67c22eefd8fe360b7617a3ea/west.yml#L21)
